scripts/generate_metrics.py

```diff
@@ -27,6 +27,7 @@
     type=str,
     default=None,
     help="The model variant (configuration) to benchmark. E.g. 7b, 13b, 70b.",
+    required=True,
```

Comment:
you need either model_variant or model_path, but you can can have architecture and model_path without variant for example

Reply:
variant seems to be required for this script, since it uses this information for creating file name prefixes etc: https://github.com/foundation-model-stack/aiu-fms-testing-utils/pull/38/files#diff-f2d7f13363f393d17bef3e70f84983fd1283ac80f0a826e92596151c753c030cL111

So I thought of making it explicit.